feature-libs/asm/components/asm-bind-cart/asm-bind-cart.component.ts

```diff
@@ -70,8 +79,12 @@ export class AsmBindCartComponent implements OnInit, OnDestroy {
   );
 
   activeCartId = '';
+  inactiveCartId = '';
+  savedInactiveCart = false;
```

Comment:
not sure we really need global variable for them.  for boolean, rename savedInactiveCart to IsinactiveCartSaved?

Reply:
for mockup, after navigate to saved cart detail we need set this save button disappear, so used this savedInactiveCart

Reply:
OK it's used for html ng-if display